placeholder-style.css

```diff
@@ -50,6 +50,14 @@ div.print:after {
     left: -2em;
 }
 
+div.howtocode {
```

Comment:
none of the previous chapters used this class, so ... stub CSS - ignore this file.